Apollo/Components/AddButton.cs

```diff
@@ -31,14 +38,92 @@ public virtual bool AlwaysShowing {
         protected virtual void Unloaded(object sender, VisualTreeAttachmentEventArgs e) => Added = null;
 
         bool mouseHeld = false;
+        
+        private DispatcherTimer _hoverTimer;
 
         protected void MouseEnter(object sender, PointerEventArgs e) {
             Fill = (IBrush)Application.Current.Styles.FindResource(mouseHeld? "ThemeButtonDownBrush" : "ThemeButtonOverBrush");
+            
+            if (_hoverTimer != null) {
+                _hoverTimer.Stop();
+            }
+            
+            if (sender is Grid grid && Preferences.UIMotion != UIMotionType.Off)
+            {
+                grid.Transitions = null;
+                bool isHorizontal = grid.ColumnDefinitions?.Count > 0;
+
+                int hoverTime = Preferences.UIHoverTime switch
+                {
+                    UIHoverTimeType.None => 0,
+                    UIHoverTimeType.Short => 100,
+                    UIHoverTimeType.Medium => 250,
+                    UIHoverTimeType.Long => 500,
+                    _ => 0
+                };
+                
+                _hoverTimer = new DispatcherTimer {
+                    Interval = TimeSpan.FromMilliseconds(hoverTime)
+                };
```

Comment:
surely there's a better way to do it than using timers... can we programmatically construct the avalonia style with animations included, as it was in xaml before, based on the settings?